<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NaoCar Remote</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="StreamDisplay" name="streamView" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnect"/>
   <addaction name="actionRift"/>
  </widget>
  <widget class="QDockWidget" name="Controls">
   <property name="minimumSize">
    <size>
     <width>215</width>
     <height>414</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Host</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>View</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="viewComboBox">
       <item>
        <property name="text">
         <string>Camera 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Camera 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>OpenCV</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Gamepad ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gamepadID">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="value">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Actions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="carambarButton">
       <property name="text">
        <string>Take Carambar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Talk</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="talkInput">
       <property name="placeholderText">
        <string>Enter text here...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Auto-driving</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="autoDrivingButton">
       <property name="text">
        <string>Launch auto driving</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnect">
   <property name="icon">
    <iconset resource="../Resources/Ui.qrc">
     <normaloff>:/connect.png</normaloff>:/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionRift">
   <property name="icon">
    <iconset resource="../Resources/Ui.qrc">
     <normaloff>:/oculus.png</normaloff>:/oculus.png</iconset>
   </property>
   <property name="text">
    <string>actionRift</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StreamDisplay</class>
   <extends>QWidget</extends>
   <header location="global">StreamDisplay.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources/Ui.qrc"/>
 </resources>
 <connections/>
</ui>
